fetch/main.js

```diff
@@ -0,0 +1,3 @@
+fetch('https://pokeapi.co/api/v2/pokemon/134')
+  .then((response) => response.json())
+  .then((vaporeon) => console.log(vaporeon));
```

Comment:
Since `fetch` requests always have a chance to fail, please make sure you're always handling that failure.